pages/common/vim.md

```diff
@@ -14,22 +14,18 @@
 
 `view {{file}}`
 
-- Switch to normal mode:
+- Save and Exit vim:
```

Comment:
I get the motivation (a beginner can easily get trapped in vim), but IMO putting the exit command first sends the wrong signals and may reinforce the beginner's apprehension. Since we're providing a short guide here, I don't think having this at the end would make it much less visible or useful. So my suggestion is leaving this at the end.